frontend/src/components/Footer.js

```diff
@@ -23,150 +21,61 @@ function Footer() {
-          Navigation
-        </Text>
-        <Text
-          fontFamily="primaryFont"
-          fontWeight="medium"
-          fontSize="md"
-          color="gray"
-          _hover={{ textDecoration: "underline", textDecorationColor: "grey" }}
-        >
-          <Link to="/">Home</Link>
-        </Text>
-        <Text
-          fontFamily="primaryFont"
-          fontWeight="medium"
-          fontSize="md"
-          color="gray"
-          _hover={{ textDecoration: "underline", textDecorationColor: "grey" }}
-        >
-          <Link to="/members">Members</Link>
-        </Text>
-        <Text
-          fontFamily="primaryFont"
-          fontWeight="medium"
-          fontSize="md"
-          color="gray"
-          _hover={{ textDecoration: "underline", textDecorationColor: "grey" }}
-        >
-          <Link to="/intern-projects">Highlights</Link>
-        </Text>
-        <Text
-          fontFamily="primaryFont"
-          fontWeight="medium"
-          fontSize="md"
-          color="gray"
-          _hover={{ textDecoration: "underline", textDecorationColor: "grey" }}
-        >
-          <Link to="/about-us">About Us</Link>
-        </Text>
-        <Text
-          fontFamily="primaryFont"
-          fontWeight="medium"
-          fontSize="md"
-          color="gray"
-          _hover={{ textDecoration: "underline", textDecorationColor: "grey" }}
-        >
-          <Link to="https://github.com/sai-nayunipati/upe-bu-cs-major-guide">
-            UPE@BU C.S. Major Guide
-          </Link>
-        </Text>
-      </VStack>
-      <VStack spacing={2} align={["center", "center", "flex-start"]}>
-        <Text
-          color="black"
-          fontFamily="secondaryFont"
-          fontWeight="bold"
-          fontSize="md"
-        >
-          Connect with us on social media!
-        </Text>
-        <HStack spacing={1}>
-          <ChakraLink
-            href="https://instagram.com/upebu"
-            color="gray"
-            fontFamily="primaryFont"
-            fontWeight="bold"
-            fontSize="md"
-          >
-            Instagram
+          Connect with Us
+        </Text>      
+        {/* instagram icon - attach link */}
+    
+        <HStack spacing={4} align="center">
+        <ChakraLink href="https://instagram.com/upebu" isExternal>
+        <Icon as={FaInstagram} boxSize="30px" />
+      </ChakraLink>
+      <HStack spacing={4} align="center">
+          <ChakraLink href="https://www.facebook.com/upeatbu/">
+          <Icon as={FaFacebook} boxSize="30px" />
           </ChakraLink>
-          <Icon as={FaInstagram} boxSize="15px" />
-        </HStack>
-        <HStack spacing={1}>
-          <ChakraLink
-            href="https://www.facebook.com/upeatbu/"
-            color="gray"
-            fontFamily="primaryFont"
-            fontWeight="bold"
-            fontSize="md"
-          >
-            Facebook
-          </ChakraLink>
-          <Icon as={FaFacebook} boxSize="15px" />
```

Comment:
As far as I know, we primarily use Instagram, LinkedIn, and email. Can you remove the other social media links? I acknowledge that this is a deficiency with the previous site.